src/backend/api/Fusion.Resources.Domain/Queries/Departments/GetRelevantOrgUnits.cs

```diff
@@ -0,0 +1,382 @@
+    /// </summary>
+    public class GetRelevantOrgUnits : IRequest<QueryRangedList<QueryRelevantOrgUnit>>
+    {
+        public GetRelevantOrgUnits(string profileId, AspNetCore.OData.ODataQueryParams query)
+        {
+            ProfileId = profileId;
+            Query = query;
+        }
+
+        /// <summary>
+        /// Mail or azure unique id
+        /// </summary>
+        public PersonId ProfileId { get; set; }
+        public ODataQueryParams Query { get; }
+        public const string CACHEKEY = "Cache.OrgUnits";
+        public enum Roles
+        {
+            DelegatedManager,
+            DelegatedParentManager,
+            ParentManager,
+            Manager,
+            Write
+        }
+
+        public class Handler : IRequestHandler<GetRelevantOrgUnits, QueryRangedList<QueryRelevantOrgUnit>>
+        {
+            private readonly TimeSpan defaultAbsoluteCacheExpirationHours = TimeSpan.FromHours(1);
+            private readonly ILogger<Handler> logger;
+            private readonly IFusionProfileResolver profileResolver;
+            private readonly IFusionRolesClient rolesClient;
+            private readonly IMediator mediator;
+            private readonly ILineOrgResolver lineOrgResolver;
+            private readonly IMemoryCache memCache;
+            private readonly HttpClient lineOrgClient;
+
+
+
+
+            public Handler(ILogger<Handler> logger, IFusionProfileResolver profileResolver, IFusionRolesClient rolesClient, ILineOrgResolver lineOrgResolver, IMediator mediator, IMemoryCache memCache, IHttpClientFactory httpClientFactory)
+            {
+                this.logger = logger;
+                this.profileResolver = profileResolver;
+                this.rolesClient = rolesClient;
+                this.mediator = mediator;
+                this.lineOrgResolver = lineOrgResolver;
+                this.memCache = memCache;
+                this.lineOrgClient = httpClientFactory.CreateClient(IntegrationConfig.HttpClients.ApplicationLineOrg());
+
+
+
+
+
+            }
+
+            public async Task<List<QueryRelevantOrgUnit>> ResolveAllOrgUnits()
+            {
+                var orgUnits = await memCache.GetOrCreateAsync(CACHEKEY, async (entry) =>
+                {
+                    entry.AbsoluteExpirationRelativeToNow = TimeSpan.FromMinutes(60);
+
+
+
+                    var orgUnitResponse = await lineOrgClient.GetFromJsonAsync<ApiPagedCollection<ApiOrgUnit>>("/org-units?$top=10000");
+                    if (orgUnitResponse is null)
+                        throw new InvalidOperationException("Could not fetch org units from line org");
+
+                    List<QueryRelevantOrgUnit> QueryorgUnits = orgUnitResponse.Value.Select(org => new QueryRelevantOrgUnit
+                    {
+                        FullDepartment = org.FullDepartment ?? "",
```

Comment:
no need to add blanks, just add the property. null is fine 👍 
As far as I remember all properties contains value as of now.